### ISSUE TITLE
Every time I upload a video I get error “Failed to get transcoded media location” once the video finishes uploading and I try to play it.

### ISSUE THREAD

**Issue by sallyFoster** [OdyseeTeam/odysee-ios#439]
![image](https://github.com/OdyseeTeam/odysee-ios/assets/104597302/3db004d4-f287-4b82-a292-9e7ed83d3ee7)

This has happened with my 3 latest video updates. All the videos that I uploaded before work fine.

Link:

https://odysee.com/@USA:27/tupac-explains-how-America-is-a-gang:9

**Comment by tzarebczan** [comment]
Does it start working at some point later? It's probably an edge case when accessed right after upload and the file isn't fully available.

**Comment by sallyFoster** [comment]
@tzarebczan No, the ones that I uploaded hours before are still showing the same error.

**Comment by tzarebczan** [comment]
We'll look into it!

**Comment by sallyFoster** [comment]
@tzarebczan Thanks!

**Comment by wavestrike1234** [comment]
Same for me
